### PR TITLE
Update Plugin - noxoua-activity-log

### PR DESCRIPTION
**Pull request by noxoua**

### CODE REVIEW

File: content/plugins/noxoua-activity-log.md [hunk 1]
```diff
@@ -3,12 +3,12 @@ name: Activity Log
 slug: noxoua-activity-log
 author_slug: noxoua
 categories: [panel-builder]
-description: Add-on for simplified activity logging based on spatie/laravel-activitylog package.
+description: Add-on for simplified activity logging based on spatie package.
 discord_url: https://discord.com/channels/883083792112300104/1167950609970888774
-docs_url: https://raw.githubusercontent.com/noxoua/filament-activity-log/main/README.md
+docs_url: https://activity-log.noxo.app/
```
Comment (awcodes) [inline content/plugins/noxoua-activity-log.md:8]: This should be 'url' instead of 'docs_url' since it is not raw markdown.

**Review by awcodes — CHANGES_REQUESTED** [review]

File: content/plugins/noxoua-activity-log.md (hunk 1 above)
Comment (noxoua) [reply content/plugins/noxoua-activity-log.md]: Okay, I would leave url to the readme file 👍

**Review by awcodes — APPROVED** [review]

**Comment by noxoua** [comment]
Author avatar replaced

**Review by saade — CHANGES_REQUESTED** [review]
Hey! Your author image must be square and have dimensions of at least 1000x1000px as per our [Contributing Guide](https://github.com/filamentphp/filamentphp.com#setting-up-an-author-profile).

Also, your plugin's image dimensions must be at least 2560x1440px as per our [Contributing Guide](https://github.com/filamentphp/filamentphp.com#submitting-a-plugin) too.

Please let us know when you've addressed these issues.

**Review by saade — APPROVED** [review]

**Pull request merged**